docs/UsersGuide.asciidoc

```diff
@@ -1010,11 +1010,15 @@ Example for `_DEPRIORITIZEBUILD` and `_DEPRIORITIZE_LIMIT`.
 
 [source,sh]
 --------------------------------------------------------------------------------
-openqa-cli api -X POST isos ISO=my_iso.iso DISTRI=my_distri FLAVOR=sweet \
-         ARCH=my_arch VERSION=42 BUILD=1234 \
+openqa-cli api -X POST isos async=0 ISO=my_iso.iso DISTRI=my_distri \
+         FLAVOR=sweet ARCH=my_arch VERSION=42 BUILD=1234 \
          _DEPRIORITIZEBUILD=1 _DEPRIORITIZE_LIMIT=120 \
 --------------------------------------------------------------------------------
 
+*NOTE* Use `async=1` to avoid possible timeouts by performing the task in background.
+This is recommended on big instances but has the disadvantage that the results (and
+possible errors) need to be polled via `openqa-cli api isos/$scheduled_product_id`.
+
```

Comment:
Please describe what the default behaviour is, corresponding to `async=0`. After that include your text. E.g.

```suggestion
*NOTE* By default scheduling products is done synchronously within the requests,
corresponding to the parameter `async=0`. Use `async=1` to avoid possible
timeouts by performing the task in background.
This is recommended on big instances but means that the results (and
possible errors) need to be polled via `openqa-cli api isos/$scheduled_product_id`.

```